### PR TITLE
[NAE-1607] Allowed nets configuration for newly created filter

### PR DESCRIPTION
**Pull request by renczesstefan**
# Description

Added allowed net field to filter processes.

Implements [NAE-1607]

## Dependencies

No new dependencies were introduced.

### Third party dependencies

No new dependencies were introduced.

### Blocking Pull requests

There are no dependencies on other PR.

## How Has Been This Tested?

This was tested manaully.

### Test Configuration

| Name                | Tested on |
|---------------------| --------- |
| OS                  |   macOS Ventura 13.1     |
| Runtime             |  Java 11        |
| Dependency Manager  |  Maven 3.8.4         |
| Framework version   |  Spring Boot 2.6.2        |
| Run parameters      |           |
| Other configuration |           |

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes have been checked, personally or remotely, with @mladoniczky 
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have resolved all conflicts with the target branch of the PR
- [x] I have updated and synced my code with the target branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes:
    - [x] Lint test
    - [x] Unit tests
    - [x] Integration tests
- [x] I have checked my contribution with code analysis tools:
    - [x] [SonarCloud](https://sonarcloud.io/project/overview?id=netgrif_components)
    - [x] [Snyk](https://app.snyk.io/org/netgrif)
- [x] I have made corresponding changes to the documentation:
    - [x] Developer documentation
    - [x] User Guides
    - [x] Migration Guides

[NAE-1607]: https://netgrif.atlassian.net/browse/NAE-1607?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/resources/petriNets/engine-processes/filter.xml [hunk 1]
```diff
@@ -206,6 +210,7 @@
         <i18nString name="missing_allowed_nets">Chýbajúce procesy</i18nString>
         <i18nString name="missing_allowed_nets_desc">Zoznam chýbajúcich procesov pre aktuálny filter</i18nString>
         <i18nString name="parent_filter_id">ID rodičovského filtra</i18nString>
+        <i18nString name="allowed_nets">Povolené siete</i18nString>
     </i18n>
     <i18n locale="de">
```
Comment (mazarijuraj) [inline src/main/resources/petriNets/engine-processes/filter.xml:215]: Missing translation of `allowed_nets` title

File: src/main/resources/petriNets/engine-processes/org_group.xml [hunk 2]
```diff
@@ -185,7 +185,8 @@
 			assignTask(initTask)
 			setData(initTask, [
 				"filter_case": ["type":"caseRef", "value": [filterAutocomplete.value], "allowedNets":["filter"]],
-				"parentId": ["type":"text", "value":useCase.stringId]
+				"parentId": ["type":"text", "value":useCase.stringId],
+				"allowed_nets": ["type": "multichoice_map", "value": filterCase.dataSet["allowed_nets"].value]
```
Comment (mazarijuraj) [inline src/main/resources/petriNets/engine-processes/org_group.xml:189]: Does it work on frontend if you set only values and it has no options?

**Review by mazarijuraj — CHANGES_REQUESTED** [review]

File: src/main/resources/petriNets/engine-processes/org_group.xml (hunk 2 above)
Comment (renczesstefan) [reply src/main/resources/petriNets/engine-processes/org_group.xml:189]: This is on developer, how he sets the option and value for this field. In filter_preference_item this field is not even editable on frontend.

File: src/main/resources/petriNets/engine-processes/filter.xml (hunk 1 above)
Comment (renczesstefan) [reply src/main/resources/petriNets/engine-processes/filter.xml:215]: Will correct it.

**Review by mazarijuraj — APPROVED** [review]

File: src/test/groovy/com/netgrif/application/engine/insurance/OrsrTest.groovy [hunk 3]
```diff
@@ -21,9 +23,12 @@ class OrsrTest {
     void parseTest() {
         def ICO = 50_903_403 as String
 
-        OrsrReference info = service.findByIco(ICO)
-
-        assertCorrectValidOrsrInfo(info)
+        try {
+            OrsrReference info = service.findByIco(ICO)
```
Comment (machacjozef) [inline src/test/groovy/com/netgrif/application/engine/insurance/OrsrTest.groovy:27]: ehmm its test... 🫠

Comment (renczesstefan) [reply src/test/groovy/com/netgrif/application/engine/insurance/OrsrTest.groovy]: Corrected.

**Review by machacjozef — APPROVED** [review]

**Pull request merged**